frontend/src/main/pages/CourseByInstructor/CourseByInstructorIndexPage.js

```diff
@@ -0,0 +1,12 @@
+
+import BasicLayout from "main/layouts/BasicLayout/BasicLayout";
+
+export default function CourseByInstructorIndexPage() {
+  return (
+    <BasicLayout>
+      <div className="pt-2">
+        <h5>Coming soon: a page where you can search by instructor</h5>
```

Comment:
See: this is where the fact that it's a placeholder is documented, basically in the code itself.  So the comment in `frontend/src/App.js` is liable to not get updated when this changes, stick around, and become misleading.